### ISSUE TITLE
Feature request: show the entire folder path instead of the folder name

### ISSUE THREAD

**Issue by syldub69** [JoseTomasTocino/QuickBookmarkToFolder#11]
Would it be possible to have an options to show the entire folder path instead of the folder name?
I often have the same folder name in many different locations (because I use the same structure of folders in the Bookmarks Toolbars and in the Bookmarks Menu).
And I have a lot of nested folders, it is not easy to know which one is the accurate one for storing the bookmark if I only see the folder name.
![2018-02-27_20-21-34](https://user-images.githubusercontent.com/16567425/36750583-3026c90e-1bfe-11e8-92ce-8ecb76a4d573.jpg)

**Comment by JoseTomasTocino** [comment]
Currently it shows "parent folder - folder", or just the folder name if it's a topmost element. Not sure about those "IBM" elements. Are they stored in the bookmark toolbar?

**Comment by syldub69** [comment]
You are right! Actually, I did not remark the "Parent folder - Folder" feature, because I already use this type of naming in my folders (as in "Hardware - Tools - IBM servers")...
The 2 IBM folders are root folders: one on the Bookmarks Toolbars and one in the Bookmarks Menu.
But the 2 could have been on the Bookmarks Toolbars, as Firefox allows to have many folder with the same name on the same level.
I think for showing the "Parent folder - Folder" structure it will be better to use the slash \ character, which is more easily interpreted as a path separator.
The best would be to have the full folder path, something like : 
"Root folder \ Level 1 folder \ Level 2 folder \ Final folder"

**Comment by yphastos** [comment]
I support this idea. I have the exact same problem, i.e. having several folders with the same name but in different parts of my whole folder tree.

e.g.

reference/php
reference/js/jQuery
tutorials/jQuery/jqGrid
utils/jQuery/tools
utils/php
utils/php/PHPExcel
etc...

as syldub69 suggests, change the hint from having hyphens '-' to '/' (or '\') indicating different folders.
And show full paths all the time, instead of only 'parentFolder-folder'.

Another cool feature would be to also highlight the matching text, and even match children folders based on a matching parent name, (i.e. not only match final folders, but any level)
e.g. if I type 'utils' it should show:

**utils**
**utils**/js
**utils**/php
**utils**/mysql

Then I could choose among all those, and not just the **utils** parent folder.

IMO it would make the add-on MUCH more useful.

**Comment by sahil48** [comment]
I would like this feature too. I would probably like the entire folder path to be shown below each entry in a smaller font, or as a pop-up tooltip when you hover over a given folder.
